homework-submission/week1-react/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import todoLogo from './todo.png';
+import './App.css';
+import Todo from "./todo.js";
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+
+          <img src={todoLogo} className="todo-logo" alt="todo" />
+          <div class="text-box">
+            <h2>Todo List</h2>
+
+            <Todo class="todo-item" item="go to Noordwijkerhout" date='2018-10-17' />
```

Comment:
In React we use `className` instead of `class` to avoid collision with `class` keyword in javascript.
Also try to style components in their respective css file, todo.js can have a css file and styled in its css file

Reply:
Also it will be better to give the props a better name, for example `description` and `deadline`